### PR TITLE
Fixes import/build for macos

### PR DESCRIPTION
**Pull request by i82orbom**

### CODE REVIEW

File: import.py [hunk 1]
```diff
@@ -13,6 +14,18 @@
 
 from strip_other_fns import strip_other_fns_and_write
 
+is_macos = True if platform.system() == "Darwin" else False
```
Comment (simonlindholm) [inline import.py:17]:
```suggestion
is_macos = (platform.system() == "Darwin")
```

File: import.py [hunk 2]
```diff
@@ -13,6 +14,18 @@
...
+
+# homebrew_gcc_cpp = max(f for f in os.listdir("/usr/local/bin") if f.startswith("cpp-"))
```
Comment (simonlindholm) [inline import.py:19]: remove this line

File: import.py [hunk 3]
```diff
@@ -145,15 +158,14 @@ def fixup_build_command(
 
     return res, assembler
 
-
```
Comment (simonlindholm) [inline import.py:148]: fwiw this newline is due to `black` auto-formatting; it'll get reinserted the next time I run that. Don't worry about it though

File: import.py [hunk 4]
```diff
@@ -13,6 +14,18 @@
...
+
+# homebrew_gcc_cpp = max(f for f in os.listdir("/usr/local/bin") if f.startswith("cpp-"))
+def homebrew_gcc_cpp():
+    try:
+        return max(f for f in os.listdir("/usr/local/bin") if f.startswith("cpp-"))
+    except ValueError:
+        return ""
+
+cpp_cmd = homebrew_gcc_cpp() if is_macos else "cpp"
+readlink_cmd = "greadlink" if is_macos else "readlink"
```
[the remark this answers is not in the document]
Comment (i82orbom) [reply import.py]: It does!

**Pull request merged**